conf/nginx.conf

```
@@ -2,15 +2,16 @@
 location __PATH__/ {
 
   # Path to source
-  alias __FINALPATH__/;
+  alias __INSTALL_DIR__/;
 
   index index.php;
  
   client_max_body_size 100M;
-
+  
+  # redirect some entire folders
   rewrite ^__PATH__/(vendor|translations|build)/.* /index.php break;
 
-  try_files $uri /index.php;
+  try_files $uri __PATH__/index.php$is_args$args;
```

`__PATH__` is always `/`, leading to ending up with `//index.php` (to slashes) which may trigger the 404 issue we're seeing

```suggestion
  try_files $uri /index.php$is_args$args;
```